indica/readers/ppfreader.py

```diff
@@ -611,14 +611,9 @@ def requires_authentication(self):
         Returns
         -------
         :
-            True if authenticationis needed, otherwise false.
+            True if authentication is needed, otherwise false.
         """
-        # Perform the necessary logic to know whether authentication is needed.
-        try:
-            self._client.list("/")
-            return False
-        except AuthenticationFailed:
-            return True
+        return self._client.auth_required
```

Comment:
There is a behaviour change here. Before, once authenticated this would return `False`, however now it will still return `True`. This effectively places remembering if you've already authenticated on the user.